### ISSUE TITLE
Adding anonymized function [Phase IV]

### ISSUE THREAD

**Issue by davidducos** [mydumper/mydumper#821]
After #724, I'm thinking that will be nice to create functions based on word files like https://github.com/dwyl/english-words/raw/master/words_alpha.txt.
In the configuration file will be nice to have: 
`random_format_with_mem <file1> '@' <file2>`
where file1 will contain the email address without the domain, and the file2 will contain different domain names. We will need to load the files to memory into lists. The files could be reuse, so we need to keep track of the filenames used.
Similar with addresses where we have a known format:
`random_format_with_mem <file_with_numbers> <file_with_streets> ',' <file_with_cities> ',' <file_with_states> ',' <file_with_countries>`
